### ISSUE TITLE
No Class defined throws Index was out of range build error

### ISSUE THREAD

**Issue by geoffreymcgill** [bridgedotnet/Bridge#983]
Create a new Class Library Project, install-package **Bridge**. 

In the generated .cs file, remove the class definition.

**Class1.cs**

```csharp
namespace ClassLibrary3
{
}
```

Throws:

```
Error		Error: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index    at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   at System.Collections.Generic.List`1.get_Item(Int32 index)
   at Bridge.Translator.Emitter.QuickSort(IList`1 list, Int32 l, Int32 r)
   at Bridge.Translator.Emitter.SortTypesByInheritance()
   at Bridge.Translator.Emitter..ctor(IDictionary`2 typeDefinitions, BridgeTypes bridgeTypes, List`1 types, IValidator validator, IMemberResolver resolver, Dictionary`2 typeInfoDefinitions, ILogger logger)
   at Bridge.Translator.Translator.CreateEmitter(IMemberResolver resolver)
   at Bridge.Translator.Translator.Translate()
   at Bridge.Build.GenerateScript.Execute()	ClassLibrary3			
```

**Comment by MangelMaxime** [comment]
Hello @geoffreymcgill , this is not the only case that generate this error.

I am working on writing a Definition Library, and I got the same error.
I followed the tutorial from here.

Here is the look of my project :
![Preview of the project](http://content.screencast.com/users/Shadaen/folders/Jing/media/15018e08-44c3-4de9-b7d0-95b064d75950/2016-03-10_2229.png)

If I try to compile the project using Bridge v1.11.2 it's generating the error.
I had to downgrade to 1.10.4 to be able to compile the library.

If I remove the `[External]` attribute I can compile the project.

**Comment by vladsch** [comment]
Hi

The issue is fixed, the pull request will be merged soon to master branch

**Comment by MangelMaxime** [comment]
Hi @vladsch glad to hear it :)

**Comment by geoffreymcgill** [comment]
I can confirm both the samples posted in this Issue are now working with the latest from **master**. The changes will be included in the 1.12 release.
